_layouts/workshop.html

```diff
@@ -9,10 +9,26 @@
   <head>
     <title>Software Carpentry: {{ page.venue }}</title>
     {% include header.html %}
-    <meta name="venue" content="{{page.venue}}" />
-    <meta name="date" content="{{page.humandate}}" />
+    <meta name="slug" content="{{site.github.project_title}}" />
     <meta name="startdate" content="{{page.startdate}}" />
     <meta name="enddate" content="{{page.enddate}}" />
+    <meta name="country" content="{{page.country}}" />
+    <meta name="venue" content="{{page.venue}}" />
+    <meta name="address" content="{{page.address}}" />
+    <meta name="latlng" content="{{page.latlng}}" />
+    <meta name="language" content="{{page.language}}" />
+    <meta name="eventbrite" content="{{page.eventbrite}}" />
+    {% if page.instructor == "FIXME" %}
```

Comment:
Hi Greg,

I tested it and it appears that the `if` conditional is not needed, and all we need is `|join:'|'`.

The outputs of `{{ var | join:'|' }}` are:
* `var = ''`: `<meta name="var" content="" />`
* `var = 'FIXME'`: `<meta name="var" content="FIXME" />`
* `var = ['Piotr Banaszkiewicz']`: `<meta name="var" content="Piotr Banaszkiewicz" />`
* `var = ['Piotr Banaszkiewicz', 'Greg Wilson']`: `<meta name="var" content="Piotr Banaszkiewicz|Greg Wilson" />`

I'll remove that `if` conditionals.